docs/source/admin/buildbootstrapguide.rst

```diff
@@ -0,0 +1,131 @@
+
+Link mysql connector to Hive::
+
+  sudo ln -s /usr/share/java/mysql-connector-java.jar /var/lib/ambari-server/resources/
+
+Place the Spark Shuffle Jar in Yarn Node Manager Folder::
+
+  sudo cp /usr/hdp/3.1.4.0-315/spark2/aux/spark-2.3.2.3.1.4.0-315-yarn-shuffle.jar /usr/hdp/current/hadoop-yarn-nodemanager/lib/.
+
+
+Grab Ambari blueprint and cluster configuration down and place in staging directory::
+
+    sudo curl -s https://location-where-blueprint-is/x-node-blueprint.json -o /home/staging/blueprint.json
+    sudo curl -s https://location-where-blueprint-is/x-node-cluster.json -o /home/staging/cluster.json
+
+Here you can learn more about Ambari `Blueprints and Cluster Configurations <https://cwiki.apache.org/confluence/display/AMBARI/Blueprints>`_
+
+Grab your bootstrap scripts::
+
+    sudo curl -s https://location-where-bootstrap-is/bootstrap.sh -o /home/staging/bootstrap.sh
+    sudo curl -s https://location-where-bootstrap-is/x-node-bootstrap.py -o /home/staging/real_bootstrap.py
+
+Optional Cloud Formation set up for bootstrap, highly recommended, OS must support Cloud Formation::
+
+  sudo curl https://location-where-bootstrap-is/bootstrap.cfg -o /etc/cloud/cloud.cfg.d/99_bootstrap.cfg
+
+
+For more information on AWS CloudFormation look `here <https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/GettingStarted.Walkthrough.html>`_ and for Azure Cloud Formation Supported OS look `here <https://docs.microsoft.com/en-us/azure/virtual-machines/linux/using-cloud-init>`_
+
+
+Grab Koverse RPMs::
+
+  sudo curl https://nexus.koverse.com/nexus/content/repositories/releases/com/koverse/koverse-server/${KOVERSE_VERSION}/koverse-server-${KOVERSE_VERSION}.rpm -o /home/staging/koverse-server-${KOVERSE_VERSION}.rpm
+  sudo curl https://nexus.koverse.com/nexus/content/repositories/releases/com/koverse/koverse-webapp/${KOVERSE_VERSION}/koverse-webapp-${KOVERSE_VERSION}.rpm -o /home/staging/koverse-webapp-${KOVERSE_VERSION}.rpm
+
+Install Koverse then clean up rpms after::
+
+    sudo yum install -y /home/staging/koverse-server-$KOVERSE_VERSION.rpm
+    sudo yum install -y /home/staging/koverse-webapp-$KOVERSE_VERSION.rpm
+    sudo rm -f /home/staging/*.rpm
+Install Koverse Ambari Stack::
+
+  sudo mkdir /var/lib/ambari-server/resources/stacks/HDP/3.1/services/KOVERSE
+  sudo cur https://koverse-bdaas.s3.amazonaws.com/hdp3/koverse-ambari-stack-3.0.tar.gz -o /var/lib/ambari-server/resources/stacks/HDP/3.1/services/KOVERSE/KOVERSE.tar.gz
+  sudo tar zxvf /var/lib/ambari-server/resources/stacks/HDP/3.1/services/KOVERSE/KOVERSE.tar.gz -C /var/lib/ambari-server/resources/stacks/HDP/3.1/services/KOVERSE
+  sudo rm -f $AMBARI_STACK_DIR/KOVERSE/KOVERSE.tar.gz
+
```

Comment:
Does it make sense to include information about setting the ulimit configuration for the koverse and accumulo users here?  I feel like that comes up consistently on systems that undergo significant load.

Reply:
Yes it does. Will integrate this